provider/image.go

```diff
@@ -43,8 +43,10 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/resource/plugin"
 )
 
-const defaultDockerfile = "Dockerfile"
-const defaultBuilder = "2"
+const (
+	defaultDockerfile = "Dockerfile"
+	defaultBuilder    = "2"
+)
```

Comment:
Some of the formatting noise here is due to `gofumpt` (the formatter we use in pu/pu) cleaning things up. I can break this out into a separate change if it's too noisy.